warehouse/query-core/src/main/java/datawave/query/iterator/QueryOptions.java

```diff
@@ -1282,17 +1281,18 @@ public boolean validateOptions(Map<String,String> options) {
         
         if (options.containsKey(PROJECTION_FIELDS)) {
             this.projectResults = true;
-            this.useWhiteListedFields = true;
             
             String fieldList = options.get(PROJECTION_FIELDS);
-            if (fieldList != null && EVERYTHING.equals(fieldList)) {
-                this.whiteListedFields = UniversalSet.instance();
-            } else if (fieldList != null && !fieldList.trim().equals("")) {
+            if (fieldList != null && !EVERYTHING.equals(fieldList) && !fieldList.trim().equals("")) {
+                this.useWhiteListedFields = true;
                 this.whiteListedFields = new HashSet<>();
                 Collections.addAll(this.whiteListedFields, StringUtils.split(fieldList, Constants.PARAM_VALUE_SEP));
-            }
-            if (options.containsKey(HIT_LIST) && Boolean.parseBoolean(options.get(HIT_LIST))) {
-                this.whiteListedFields.add(JexlEvaluation.HIT_TERM_FIELD);
+                if (options.containsKey(HIT_LIST) && Boolean.parseBoolean(options.get(HIT_LIST))) {
+                    this.whiteListedFields.add(JexlEvaluation.HIT_TERM_FIELD);
+                }
+            } else {
+                this.useBlackListedFields = true;
+                this.blackListedFields = new HashSet<>();
```

Comment:
I don't think you need this.  blackListedFields initial value is `new HashSet<>()`  It's actually set on line 1310 below as well.  that wasn't your addition, but that can also be removed.

Reply:
ah, ok.  Thanks.